src/activities/Elsa.Activities.Http/Activities/WriteHttpResponse.cs

```diff
@@ -106,6 +111,17 @@ public WorkflowExpression<string> ResponseHeaders
             set => SetState(value);
         }
 
+
+        /// <summary>
+        /// Variable name for model body - Read model body from named variable. 
+        /// </summary>
+        [ActivityProperty(Hint = "Variable name for content model - Read content model from named variable.")]
+        public string InputVariable
```

Comment:
We don't need this property - Variables can be retrieved via expressions. E.g. `MyVariableName`.

See also: https://elsa-workflows.github.io/elsa-core/docs/expressions-javascript (I added this documentation today, so I don't blame you for not knowing)

Reply:
I dont like that... This way my objects get mangeled twice... Once for reading and once for writing the object. Its not needed and costs a lot of CPU. How about a new expression type called Variable?